drake/systems/plants/xmlUtil.cpp

```diff
@@ -87,13 +89,25 @@ void originAttributesToTransform(tinyxml2::XMLElement* node,
 
 void poseValueToTransform(tinyxml2::XMLElement* node, const PoseMap& pose_map,
                           Eigen::Isometry3d& T,
-                          const Eigen::Isometry3d& T_default_frame) {
+                          const Eigen::Isometry3d& T_default_frame,
+                          bool printMsgs) {
   Eigen::Vector3d rpy = Eigen::Vector3d::Zero(), xyz = Eigen::Vector3d::Zero();
   const char* strval = node->FirstChild()->Value();
   if (strval) {
     std::stringstream s(strval);
     s >> xyz(0) >> xyz(1) >> xyz(2) >> rpy(0) >> rpy(1) >> rpy(2);
   }
+
+  if (printMsgs) {
```

Comment:
I agree. Until Drake has an [official logging system](https://github.com/RobotLocomotion/drake/issues/1895) that can effectively remove debug statements at the preprocessor stage, I will remove all debug statements in PRs to the master.